packages/application-components/src/utils/default-form-buttons.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { useIntl, MessageDescriptor, IntlShape } from 'react-intl';
+import {
+  PrimaryButton,
+  SecondaryButton,
+  IconButton,
+  BinLinearIcon,
+} from '@commercetools-frontend/ui-kit';
+import buttonMessages from './button-messages';
+import filterDataAttributes from './filter-data-attributes';
+
+type Label = string | MessageDescriptor;
+type Props = {
+  label: Label;
+  onClick: (event: React.SyntheticEvent) => void;
+  isDisabled: boolean;
+  dataAttributes: { [key: string]: string };
+  children?: never;
+};
+
+const commonDefaultProps = {
+  isDisabled: false,
+  dataAttributes: {},
+};
+
+const confirmDefaultProps: Pick<
+  Props,
+  'label' | 'isDisabled' | 'dataAttributes'
+> = {
+  label: buttonMessages.confirm,
+  ...commonDefaultProps,
+};
+
+const getFormattedLabel = (label: Label, intl: IntlShape) =>
+  typeof label === 'string' ? label : intl.formatMessage(label);
+
+const ConfirmButton = (props: Props) => {
```

Comment:
I would name this `PrimaryButton`

Reply:
`FormPrimaryButton` maybe so it's

1. Not confused with UIKit's `PrimaryButton`
2. Mapps nicely to the `isPrimaryButtonDisbabled`-prop

Hope that makes sense.